asynction/exceptions.py

```diff
@@ -38,3 +38,28 @@ class MessageAckValidationException(ValidationException):
     """
 
     pass
+
+
+class SecurityException(AsynctionException):
```

Comment:
```suggestion
class SecurityException(AsynctionException, ConnectionRefusedError):
```
But not sure about this one.